src/form/post-procedures/post-procedure-form.component.tsx

```diff
@@ -194,6 +218,12 @@ const PostProcedureForm: React.FC<PostProcedureFormProps> = ({
           isLowContrast: true,
           kind: "success",
         });
+      }
+      mutate(
+        (key) => typeof key === "string" && key.startsWith("/ws/rest/v1/order"),
```

Comment:
If possible we could try mutate only the procedures endpoint, this is because mutating the entire orders endpoint is expensive
```suggestion
        (key) => typeof key === "string" && key.startsWith("/ws/rest/v1/order/orderTypes=${config.procedureOrderTypeUuid}"),
```